src/app/components/Sidebar/Sidebar.jsx

```diff
@@ -85,6 +90,7 @@ Sidebar.propTypes = {
   listOptions: PropTypes.object,
   selectedFilters: PropTypes.arrayOf(PropTypes.object),
   displayInfo: PropTypes.object,
+  type: PropTypes.string,
```

Comment:
We miss the type validation for `currentType`

Reply:
Changed to type -- currentType is no longer the prop name.